protos/SignalService.proto

```diff
@@ -48,6 +49,17 @@ message LokiAddressMessage {
   optional Type   type          = 3;
 }
 
+message PairingAuthorisationMessage {
+  enum Type {
+    PAIRING_REQUEST = 1;
+    UNPAIRING_REQUEST = 2;
+  }
+  optional string primaryDevicePubKey   = 1;
```

Comment:
Not a suggestion but what is the rationale behind signal making everything in the protobuf optional?

Reply:
It looks more like an oversight than a rationale.